src/SourceBuild/content/test/Microsoft.DotNet.SourceBuild.SmokeTests/assets/LicenseExclusions.txt

```diff
@@ -0,0 +1,180 @@
+src/runtime/src/libraries/System.Text.RegularExpressions/tests/FunctionalTests/AttRegexTests.cs|other-permissive
+
+# Reference to a license, not applicable to source
+src/runtime/src/libraries/System.Text.Json/roadmap/images/core-components.txt|unknown-license-reference
+src/runtime/src/libraries/System.Text.Json/roadmap/images/higher-level-components.txt
+
+# Sample data
+src/runtime/src/mono/sample/wasm/browser-webpack/package-lock.json
+
+#
+# source-build-externals
+#
+
+# False positive
+src/source-build-externals/src/abstractions-xunit/README.md|free-unknown
+src/source-build-externals/src/application-insights/NETCORE/ThirdPartyNotices.txt|unknown
+src/source-build-externals/src/azure-activedirectory-identitymodel-extensions-for-dotnet/benchmark/Microsoft.IdentityModel.Benchmarks/CreateTokenTests.cs|proprietary-license
+src/source-build-externals/src/azure-activedirectory-identitymodel-extensions-for-dotnet/src/Microsoft.IdentityModel.JsonWebTokens/JsonClaimValueTypes.cs|proprietary-license
+src/source-build-externals/src/azure-activedirectory-identitymodel-extensions-for-dotnet/src/Microsoft.IdentityModel.Tokens.Saml/Saml/ClaimProperties.cs|proprietary-license
+src/source-build-externals/src/azure-activedirectory-identitymodel-extensions-for-dotnet/src/Microsoft.IdentityModel.Tokens.Saml/Saml/SamlTokenUtilities.cs|proprietary-license
+src/source-build-externals/src/azure-activedirectory-identitymodel-extensions-for-dotnet/src/Microsoft.IdentityModel.Tokens.Saml/Saml2/ClaimProperties.cs|proprietary-license
+src/source-build-externals/src/azure-activedirectory-identitymodel-extensions-for-dotnet/src/System.IdentityModel.Tokens.Jwt/JsonClaimValueTypes.cs|proprietary-license
+src/source-build-externals/src/humanizer/readme.md|free-unknown
+src/source-build-externals/src/humanizer/NuSpecs/*.nuspec
+src/source-build-externals/src/xunit/README.md|free-unknown
+src/source-build-externals/src/xunit/src/xunit.assert/Asserts/README.md|free-unknown
+
+#
+# source-build-reference-packages
+#
+
+# Contains references to licenses which are not applicable to the source
+src/source-build-reference-packages/src/packageSourceGenerator/PackageSourceGeneratorTask/RewriteNuspec.cs|unknown-license-reference,ms-net-library-2018-11
+
+# API only
+src/source-build-reference-packages/src/referencePackages/**
+src/source-build-reference-packages/src/targetPacks/**
+src/source-build-reference-packages/src/textOnlyPackages/**
+
+#
+# sourcelink
+#
+
+# False positive
+src/sourcelink/docs/GitSpec/GitSpec.md|unknown-license-reference
+
+#
+# test-templates
+#
+
+# Not applicable to source
+src/test-templates/Templates/**/*.vstemplate
+
+#
+# vstest
+#
+
+# False positive
+src/vstest/src/Microsoft.TestPlatform.Extensions.BlameDataCollector/ProcDumpArgsBuilder.cs|proprietary-license
+
+# Build asset, but not applying to code
+src/vstest/src/package/Microsoft.CodeCoverage/ThirdPartyNoticesCodeCoverage.txt
```

Comment:
This contains only one license (mono.cecil), and the the license looks like a standard MIT license. Any reason to add to to the exclude list here? Is it because of the preamble?